ISHMapboxDynamicFontObserver.m

```diff
@@ -134,13 +137,13 @@ + (MGLStyleValue *)styleConstantValue:(MGLConstantStyleValue *)constantValue mul
 + (MGLStyleValue *)styleCameraValue:(MGLCameraStyleFunction *)cameraValue multipliedByScale:(CGFloat)scale {
     __block NSMutableDictionary<id, MGLStyleValue *> *newStops = [NSMutableDictionary dictionaryWithCapacity:cameraValue.stops.count];
 
-    [cameraValue.stops enumerateKeysAndObjectsUsingBlock:^(id key, MGLStyleValue *value, BOOL * stop) {
-        newStops[key] = [self styleValue:cameraValue.stops[key] multipliedByScale:scale];
+    [cameraValue.stops enumerateKeysAndObjectsUsingBlock:^(id key, MGLStyleValue *value, BOOL *stop) {
+        newStops[key] = [self styleValue:value multipliedByScale:scale];
     }];
 
     MGLCameraStyleFunction *newValue = (MGLCameraStyleFunction *)[MGLStyleValue valueWithInterpolationMode:cameraValue.interpolationMode
                                                                                                cameraStops:newStops
-                                                                                                options:nil];
+                                                                                                   options:cameraValue.options];
```

Comment:
`options` is not available https://github.com/mapbox/mapbox-gl-native/blob/ios-v3.6.2/platform/darwin/src/MGLStyleValue.h#L351